python/ray/dashboard/client/src/pages/dashboard/features/Host.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { makeFeature } from "./makeFeature";
+
+const Host = makeFeature({
+  getFeatureForNode: ({ node }) => (
+    <React.Fragment>
+      {node.hostname} ({node.ip})
+    </React.Fragment>
+  ),
+  getFeatureForWorker: ({ worker }) => (
+    <React.Fragment>
+      {worker.cmdline[0].split("::", 2)[0]} (PID: {worker.pid})
```

Comment:
Can you add a comment saying `.split("::")` because ray worker proctitle convention being ray::worker or ray::ActorName?

Reply:
I added a comment to this effect.